### DISCUSSION TITLE
🎉工业时代2高版本重生计划

### DISCUSSION THREAD

**Discussion by AbbottMc** [IndustrialCraftMC/IndustrialCraft2#2] [【公告】Announcements]
👋Hello各位工业2玩家们大家好！

其实早在1.15版本时代IC2 Dev Team就已经开始尝试将工业2移植到高版本了。但由于各方面因素的影响，导致工业2高版本移植计划迟迟无法完成。

但其实目前为止很多工作都已经做完了，距离可玩版本其实就差“一步之遥”。

目前除去未迁移的物品方块外，已有的内容也有一些bug亟待修复。所以在这里我们希望社区里对工业时代2仍然感兴趣的玩家们能参与进来帮助测试（**不建议在生存模式下进行测试**，因为目前还不是可玩版本），并在反馈区提交相关问题。助力工业时代2高版本的迁移开发。

我们会在后续的开发过程中逐一排查相关问题并尝试修复。

感谢各位的支持！

👋Hello, all IC2 players!

In fact, as early as version 1.15, the IC2 Dev Team has already begun to try to port IC2 to a higher version. However, due to the influence of various factors, the porting plan has been unable to be completed.

But in fact, a lot of work has been done so far, and the playable version is actually "one step away".

In addition to the items and blocks that have not been ported, there are also some bugs in the existing content that need to be fixed urgently. So here we hope that players in the community who are still interested in IC2 can participate and help test (**it is not recommended to test under survival mode**, because it's not a playable version yet), and submit related questions in the feedback area. Assist in the porting and development of the high version of IC2.

We will troubleshoot related issues one by one and try to fix them in the subsequent development process.

Thank you for your support!

**Comment by SoraChan-uwu** [comment]
Thank you for continuing to port the mod! Good luck to you ❤️

**Comment by luwangwen** [comment]
Thank you verymuch. I try to make a modpack about ic2 and gtceu. I hope the plan succeed early

**Comment by TimelightXi** [comment]
什么时候更新IC21.20.1版本?

**Reply by TimelightXi** [reply]
不能再等了！
